armadillo/src/main/resources/application.yml

```diff
@@ -31,7 +31,6 @@ spring:
       max-request-size: 1000MB
   application:
     name: armadillo
-  security:
```

Comment:
this doesn't make sense to me?

Reply:
The whole file seems too much somehow. Logging is mixed with effort for PR #479

Reply:
Restored `security:`